packages/react-positioning/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@fluentui/react-positioning",
+  "version": "9.0.0-alpha.0",
+  "description": "A react wrapper around Popper.js for Fluent UI",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
```

You can probably get rid of a couple of these
```suggestion
    "build": "just-scripts build",
    "clean": "just-scripts clean",
    "code-style": "just-scripts code-style",
    "just": "just-scripts",
    "lint": "just-scripts lint",
    "start-test": "just-scripts jest-watch",
    "test": "just-scripts test",
    "update-snapshots": "just-scripts jest -u"
```